src/main/resources/db/migration/V0.0.3__create_print_table.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE
+    IF NOT EXISTS "tbl_print" (
+        "print_id" VARCHAR(36) UNIQUE,
+        "ygojson_id" VARCHAR(36) UNIQUE,
+        "set_id" VARCHAR(36),
+        "card_id" VARCHAR(36),
+        "print_code" TEXT,
+        "setcode" TEXT,
+        "print_number_prefix" TEXT,
+        "print_number" TEXT,
+        "print_number_suffix" TEXT,
+        "rarity" TEXT,
+        "language" TEXT,
+        "region_code" TEXT,
+        -- TODO: decide the cascading mode
```

Comment:
To do so, we should decide how we are going to use the entities. We should open a ticket to track this.